definitions/pe.yaml

```diff
@@ -0,0 +1,119 @@
+# Peruvian holiday definitions for the Ruby Holiday gem.
+#
+# created: 2016-06-18.
+# updated: 2016-06-18.
+#
+# Sources:
+# - http://www.timeanddate.com/holidays/peru/
+# - https://en.wikipedia.org/wiki/Public_holidays_in_Peru
+#
+# Avoided days:
```

Comment:
Because they are on the resources that i mencioned before but they actually are not holidays here or they aren't even informal holidays.